3-semantic-html/basic.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <nav>
+<ul>
+    <li>
+        <a href="">Home</a>
+    </li>
+    <li>
+        <a href="#artistas">Artistas</a>
+    </li>
+    <li>      
+        <a href="#digital">Servicios digitales</a>
+    </li>
+    <li>
+        <a href="">Eventos</a>
+    </li>
+</ul>
+        </nav>
+    </header>
+
+    <h2 id="artistas">Artistas</h2>
+
+    <article>
+        <h2>JBalvin</h2>
+        <img src="" alt="">
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Numquam ex maiores porro delectus! Facilis, impedit ab, incidunt sint quia corporis adipisci et atque quos obcaecati voluptates corrupti nobis quisquam explicabo.</p>
+
+    </article>
+    <article>
+        <h2>Metallica</h2>
+        <img src="" alt="">
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Numquam ex maiores porro delectus! Facilis, impedit ab, incidunt sint quia corporis adipisci et atque quos obcaecati voluptates corrupti nobis quisquam explicabo.</p>
+    </article>  
+    <article>
+        <h2>The Strokes</h2>
+        <img src="" alt="">
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Numquam ex maiores porro delectus! Facilis, impedit ab, incidunt sint quia corporis adipisci et atque quos obcaecati voluptates corrupti nobis quisquam explicabo.</p>
+    </article>  
+    <article>
+        <h2>The Do</h2>
+        <img src="" alt="">
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Numquam ex maiores porro delectus! Facilis, impedit ab, incidunt sint quia corporis adipisci et atque quos obcaecati voluptates corrupti nobis quisquam explicabo.</p>
+    </article> 
+
+    <main>
+        <h2 id="digital">Servicios Digitales</h2>
+
+        <article>
+
+            <header>
+                <h1>Mensajes personalizados</h1>
```

Comment:
Remember to order the headings tags, you have one `<h2>` above the `<h1>`. A good practice it's to set only one `<h1>` per page.